### ISSUE TITLE
Widget is of variable width

### ISSUE THREAD

**Issue by thanpolas** [attheodo/Pingu#3]
The widget is of variable width as the ping time number changes. It makes it a bit awkward to watch and pushes all other widgets left and right when it's loaded before them.

![Screen Capture on 2020-04-06 at 10-40-33](https://user-images.githubusercontent.com/458813/78545095-1ae20600-77f3-11ea-9b11-ef39a77391ba.gif)

**Comment by attheodo** [comment]
@thanpolas yeah the widget is adjusting its width to fit it's label. Imagine a response being 1001ms for example. One solution might be to use an average-width label and adjust the font size to fit. Will take a look, ty 👍
